geonode/base/api/views.py

```diff
@@ -1517,11 +1517,25 @@ def base_linked_resources_payload(instance, user, params={}):
         linked_by_visib_ids = linked_by_visib.values_list("id", flat=True)
         linked_by = [lres for lres in linked_by_over_loopable if lres.source.id in linked_by_visib_ids]
 
-        ret["linked_by"] = LinkedResourceSerializer(
-            instance=linked_by, serialize_source=True, embed=True, many=True
-        ).data
```

Comment:
is there any reason why the serializer is no longer used?

Reply:
It **is** used indeed.
Please note that the code you're pointing to has changed name from `base_linked_resources_payload` to  `base_linked_resources_instances`.  The new `base_linked_resources_payload` calls `base_linked_resources_instances` (now also used elsewere for DRY), and still uses the serializer, returning the same content as before.